src/services/database/serverless.yml

```diff
@@ -0,0 +1,77 @@
+service: ${self:custom.project}-database
+
+frameworkVersion: "3"
+
+package:
+  individually: true
+
+plugins:
+  - serverless-stack-termination-protection
+  - "@stratiformdigital/serverless-s3-security-helper"
+  - "@stratiformdigital/serverless-iam-helper"
+  - serverless-bundle
+
+provider:
+  name: aws
+  region: us-east-1
+  iam:
+    role:
+      path: /delegatedadmin/developer/
+      permissionsBoundary: arn:aws:iam::${aws:accountId}:policy/cms-cloud-admin/developer-boundary-policy
+      statements:
+        - Effect: Allow
+          Action:
+            - logs:CreateLogGroup,
+            - logs:CreateLogStream,
+            - logs:PutLogEvents,
+            - dynamodb:PutItem,
+            - dynamodb:UpdateItem,
+            - dynamodb:DeleteItem"
+          Resource: "*"
+  stackTags:
+    PROJECT: ${self:custom.project}
+    SERVICE: ${self:service}
+
+custom:
+  project: ${env:PROJECT}
+  fileHash: ${file(./items.ts)}
+  serverlessTerminationProtection:
+    stages: # Apply CloudFormation termination protection for these stages
+      - master
+      - val
+      - production
+  tableName: ${self:custom.project}-${self:service}-${sls:stage}
```

Comment:
project is already part of the service so you prob dont need that

Reply:
I have removed project.